test/README.md

```diff
@@ -1,10 +1,19 @@
-# Testing Directory
+# E2E Testing
 
-## File Directory
-- `/cloudformation`: Testing IAM Roles, Managed Prometheus Workspace, Managed Grafana Workspace
-- `/suites`: Directories defining test suites
-- `/pkg`: Common utilities and expectations
-- `/hack`: Testing scripts
+Karpenter leverages Github Actions to run our E2E test suites. These suites are triggered by:
+1. Periodic schedule runs every 8 hours
+2. New commits to the `main` branch
+3. `/karpenter snapshot` comments by maintainers on Pull Requests
```

Comment:
nit: might be nice to delineate this so that users don't comment on the PR incorrectly.
```suggestion
3. `/karpenter snapshot` review comments by maintainers on Pull Requests
```